lightkurve/lightcurve.py

```diff
@@ -131,7 +131,7 @@ def append(self, others):
                 new_lc.centroid_row = np.append(new_lc.centroid_row, others[i].centroid_row)
         return new_lc
 
-    def flatten(self, window_length=101, polyorder=3, return_trend=False, **kwargs):
+    def flatten(self, window_length=101, polyorder=2, return_trend=False, **kwargs):
```

Comment:
Note: this may change the behavior of existing tutorials.  But I guess it is more robust?

Reply:
You're right, I can put it back, but I think it might be a little high?